content/docs/get-started/aws/deploy-changes.md

```diff
@@ -12,57 +12,369 @@ menu:
 aliases: ["/docs/quickstart/aws/deploy-changes/"]
 ---
 
-Now let's deploy our changes.
+Now let's deploy your changes.
 
 ```bash
 $ pulumi up
 ```
 
-Pulumi computes the minimally disruptive change to achieve the desired state described by the program.
+First Pulumi will run the `preview` step of the update. In the future you can run `pulumi up -y` to automatically run the update after a successful preview.
 
 ```
 Previewing update (dev):
```

Comment:
don't remember where we landed on this, but do you think we should add the permalink changes to the output that Jamie added in https://github.com/pulumi/pulumi/pull/5222? Not sure what the timing will be like, i.e. if it will make it in before or after this PR.

Reply:
I think we landed where we would add the link when Jamie's changes are live. Also so we can isolate the two changes and specifically measure the impact of both.